tests/testAutopas/tests/selectors/tuningStrategy/BayesianSearchTest.cpp

```diff
@@ -0,0 +1,57 @@
+/**
+ * @file BayesianSearchTest.cpp
+ * @author Jan Nguyen
+ * @date 12.06.19
+ */
+
+#include "BayesianSearchTest.h"
+#include <gmock/gmock-matchers.h>
+#include <gmock/gmock-more-matchers.h>
+
```

Comment:
would a unit test make sense where you add some evidence and then expect a certain outcome?

Reply:
I added a test, which assigns a dummy time for each configuration. The BayesianSearch should find the constructed optimum via tuning. I also fixed the seed of the RNG to avoid random fails.

https://github.com/AutoPas/AutoPas/blob/691599b696e17a59afa7e650ad40d260e6ee08ee/tests/testAutopas/tests/selectors/tuningStrategy/BayesianSearchTest.cpp#L77-L103